setup.cfg

```diff
@@ -0,0 +1,8 @@
+[flake8]
+max-line-length = 110
+ignore = E133, E226, E228, N802, N803, N806
+exclude = __init__.py
```

Comment:
This config differs a bit from the one in the [style guide](https://developer.lsst.io/coding/python_style_guide.html#flake8-configuration-files) -- in particular, the N806 ignore is new.

Reply:
Not for science pipelines: https://developer.lsst.io/coding/python_style_guide.html#exceptions-to-pep-8

Reply:
E251 should in theory be included but in this case it never triggers anything so I left it out.

Reply:
Fine. Maybe there's a clearer way to organize the rules...